src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EnterBreathingSpaceCuiCallbackHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
I wonder whether instead of calling it CUI we could call it LIP? because in theory that handler could be called from some other program. but it is to do with breathing space in LIP case. what do you think?

Reply:
Updated to LIP instead using CUI